docs/contact-us.md

```diff
@@ -3,7 +3,7 @@ Contact Us
 
 Contact us:
 
-1. Using the user and contributor real time [Gitter chat](https://gitter.im/OpenMS/OpenMS).
+1. On the user and contributor real time [Gitter chat](https://gitter.im/OpenMS/OpenMS).
```

Comment:
Should we change this to discord? I think it is newer and more powerful and is becoming our default chat.

Reply:
It was discussed, idk, discord has a limit of free messages, something like Slack. Gitter can be thought of as **the support chat**.

I missed adding discord; it would be(should be) a prime source of contact for contributors, say: searching in past channel messages and doing things.